src/Fabulous/ViewElement.fs

```diff
@@ -155,6 +164,15 @@ type ViewElement internal (targetType: Type, create: (unit -> obj), update: (Vie
 
     /// Differentially update the inherited attributes of a visual element given the previous settings
     member x.UpdateInherited(prevOpt: ViewElement voption, curr: ViewElement, target: obj) = update prevOpt curr target
+    
+    member x.KeyValue
+            with get() = key
+            and set v = key<-v
+    member x.Key _ =x
```

Comment:
That is a requirement of code gen, we cannot put a attribute on ViewElement, only on Control itself, so we should "mock" the behavior for code to be compiled. Key  should be immutable, yes. i'll fix it

Reply:
Since `Key` has been declared as `canBeUpdated: false` in the JSON file, CodeGen won't attempt to apply the value anywhere.
It will just only keep it in the `attribs` array passed as a constructor parameter.

So `x.Key _ = x` has no use and can be safely removed.

Reply:
Oh, my bad

Reply:
x.KeyValue was deleted but not x.Key?

Reply:
Removed, it is unused now, you are right